test-files/ui/features/check_layer_visbility.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Check Layer Visibility
+
+    Scenario: Open Hootenanny and Add Data
+        Given I am on Hootenanny
+        And I click Get Started
+        And I press "Add Reference Dataset"
+        And I click the "DcGisRoadsCucumber" Dataset
+        And I press "Add Layer"
+        Then I wait 15 "seconds" to see "span.strong" element with text "DcGisRoadsCucumber"
+        And I press "Add Secondary Dataset"
+        And I click the "DcOsmRoadsCucumber" Dataset
+        And I press "Add Layer"
+        Then I wait 15 "seconds" to see "span.strong" element with text "DcOsmRoadsCucumber"
+
+    Scenario: Check Layer Visbility
+        When I click the Background settings button
+        And I should see checkbox "DcGisRoadsCucumber" checked
+        And I should see checkbox "DcOsmRoadsCucumber" checked
+        And I hover over "#map"
+        When I select a way map feature with class "w28_53"
```

Comment:
We should omit the _mapid from the class name here and below.  The mapid of the dataset may not always be 53, so the custom step is set up to wildcard it.